src/features/integration/IntegrationConfigurationForm.tsx

```diff
@@ -64,77 +68,106 @@ const useStyles = makeStyles((theme: Theme) =>
     })
 );
 
-const accordionList: IAccordion[] = [
-    {summary: "Integrasjonslogikk", accordionForm: ACCORDION_FORM.CASE_INFORMATION, defaultExpanded: true},
-    {summary: "Sakspost", accordionForm: ACCORDION_FORM.CASE_FORM, defaultExpanded: false},
-    {summary: "Journalpost", accordionForm: ACCORDION_FORM.RECORD_FORM, defaultExpanded: false},
-    {summary: "Dokument- og objektbeskrivelse", accordionForm: ACCORDION_FORM.DOCUMENT_FORM, defaultExpanded: false},
-    {summary: "Avsender", accordionForm: ACCORDION_FORM.APPLICANT_FORM, defaultExpanded: false}
-]
-
 const IntegrationConfigurationForm: React.FunctionComponent<RouteComponentProps<any>> = () => {
     const classes = useStyles();
-    const {handleSubmit, watch, setValue} = useForm<IFormData>({
-        defaultValues: defaultValues
+    const location = useLocation();
+    const [submitSuccess, setSubmitSuccess] = useState(false)
+    let activeConfiguration = location.state ? location.state as IIntegrationConfiguration : undefined;
+    let activeFormData = location.state ? toFormData(location.state as IIntegrationConfiguration) : defaultValues;
```

Comment:
This is set in the overview detailts component and kept in location state for now,  added separate task to put activeConfiguration in a context instead of location
https://fintlabs.atlassian.net/jira/software/projects/FST/boards/25/backlog?selectedIssue=FST-120

Reply:
Enig